app/views/users/show.html.erb

```diff
@@ -1,3 +1,8 @@
+<% if @user == current_user %>
+  <%= link_to 'Edit', edit_user_path(@user) %>
+  <%= link_to "Create Tweet", new_tweet_path %>
```

Comment:
It doesn't look like we have a preference of single quotes or double quotes in the style guide; however, you should try to stay consistent in whatever you (or your team) choose (lines 2 and 3 here)

http://wiki.smashingboxes.com/articles/backend